Gemfile

```diff
@@ -3,13 +3,13 @@ source "https://rubygems.org"
 ruby "2.5.7"
 
 gem "pg"
-gem "rails", "5.1.6.2"
+gem "rails", "5.2.4.2"
```

Comment:
Have you use bin/rails app:update task during update the rails version?

Reply:
You can find the update docs [here](https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html)

Reply:
sorry, now done